sdk/client/msgraph/client.go

```diff
@@ -0,0 +1,92 @@
+package msgraph
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/hashicorp/go-azure-sdk/sdk/client"
+	"github.com/hashicorp/go-azure-sdk/sdk/environments"
+)
+
+var _ client.BaseClient = &Client{}
+
+type ApiVersion string
+
+const (
+	Version10   ApiVersion = "v1.0"
```

Comment:
whilst probably not as clean, for consistency the Pandora generated SDKs are outputting these as:

```suggestion
	VersionOnePointZero   ApiVersion = "v1.0"
```

might be worth doing the same here for consistency?

Reply:
Good shout, that is better